internal/consensus/state_test.go

```diff
@@ -109,12 +110,11 @@ func TestStateProposerSelection0(t *testing.T) {
 // Now let's do it all again, but starting from round 2 instead of 0
 func TestStateProposerSelection2(t *testing.T) {
 	config := configSetup(t)
+	logger := log.TestingLogger()
```

Comment:
I might try to more explicitly use the NopLogger rather than the testing logger.